otx/api/usecases/exportable_code/demo/requirements.txt

```diff
@@ -1,4 +1,4 @@
 openvino==2022.3.0
 openmodelzoo-modelapi==2022.3.0
-otx @ git+https://github.com/openvinotoolkit/training_extensions/@2663e4ad1f4819d1de8b32ff77e0b24cc803b9c6#egg=otx
+otx @ git+https://github.com/openvinotoolkit/training_extensions/@934aa72fd71da84231701560d497ba60f399232f#egg=otx
```

Comment:
Will this change after release otx1.3?

Reply:
Yes, on every release we use pip version here, but at the development stage this commit is fixed according to the latest update of exportable code.